### PR TITLE
Fixes a bunch of \red and \blue

### PR DESCRIPTION
**Pull request by monster860**
A lot of \red has been replaced with <span class='warning'> and a lot of \blue has been replaced with <span class='notice'>

We go from 1600 instances to about 200 here with a short little script.

Only lines that have exactly one instance of \red and \blue and don't have \black and have 2 unescaped quotes are modified. This leaves 200 unmodified.

Anyways, @tigercat2000 will love this.

### CODE REVIEW

**Comment by Tastyfish** [comment]
Instances of `\red <B>` should actually become `<span class='danger'>`

**Comment by monster860** [comment]
Ow wow, travis-ci actually gave up on this?

**Comment by tigercat2000** [comment]
lol

**Comment by TheDZD** [comment]
Oh god, merge conflicts.

**Comment by monster860** [comment]
Merge conflicts fixed **again**

File: code/_onclick/item_attack.dm [hunk 1]
```diff
@@ -149,13 +149,13 @@
 
 		for(var/mob/O in viewers(messagesource, null))
 			if(attack_verb.len)
-				O.show_message("\red <B>[M] has been [pick(attack_verb)] with [src][showname] </B>", 1)
+				O.show_message("<span class='warning'><B>[M] has been [pick(attack_verb)] with [src][showname] </B></span>", 1)
```
Comment (TheDZD) [inline code/_onclick/item_attack.dm]: Should be the danger span, then just remove the bold.

File: code/datums/mind.dm [hunk 2]
```diff
@@ -635,15 +635,15 @@
 				log_admin("[key_name(usr)] has given [key_name(current)] a loyalty implant")
 				message_admins("[key_name_admin(usr)] has given [key_name_admin(current)] a loyalty implant")
 
-				H << "\red <Font size =3><B>You somehow have become the recepient of a loyalty transplant, and it just activated!</B></FONT>"
+				H << "<span class='warning'><Font size =3><B>You somehow have become the recepient of a loyalty transplant, and it just activated!</B></FONT></span>"
```
Comment (TheDZD) [inline code/datums/mind.dm]: Userdanger should do font size 3, red, and bold text.

File: code/datums/mind.dm [hunk 3]
```diff
@@ -818,7 +818,7 @@
 					special_role = "Wizard"
 					//ticker.mode.learn_basic_spells(current)
 					ticker.mode.update_wiz_icons_added(src)
-					current << "<B>\red You are the Space Wizard!</B>"
+					current << "<B><span class='warning'>You are the Space Wizard!</B></span>"
```
Comment (TheDZD) [inline code/datums/mind.dm]: Danger

File: code/datums/mind.dm [hunk 4]
```diff
@@ -984,7 +984,7 @@
 				if(src in ticker.mode.traitors)
 					ticker.mode.traitors -= src
 					special_role = null
-					current << "\red <FONT size = 3><B>You have been brainwashed! You are no longer a traitor!</B></FONT>"
+					current << "<span class='warning'><FONT size = 3><B>You have been brainwashed! You are no longer a traitor!</B></FONT></span>"
```
Comment (TheDZD) [inline code/datums/mind.dm]: Userdanger

File: code/datums/mind.dm [hunk 5]
```diff
@@ -1001,7 +1001,7 @@
 					slaved.masters += src
 					src.som = slaved //we MIGT want to mindslave someone
 					special_role = "traitor"
-					current << "<B>\red You are a traitor!</B>"
+					current << "<B><span class='warning'>You are a traitor!</B></span>"
```
Comment (TheDZD) [inline code/datums/mind.dm]: Danger

File: code/datums/mind.dm [hunk 6]
```diff
@@ -1078,7 +1078,7 @@
 					qdel(A.malf_picker)
 					A.show_laws()
 					A.icon_state = "ai"
-					A << "\red <FONT size = 3><B>You have been patched! You are no longer malfunctioning!</B></FONT>"
+					A << "<span class='warning'><FONT size = 3><B>You have been patched! You are no longer malfunctioning!</B></FONT></span>"
```
Comment (TheDZD) [inline code/datums/mind.dm]: Userdanger

File: code/game/dna/genes/goon_disabilities.dm [hunk 7]
```diff
@@ -377,7 +377,7 @@
 	if (istype(usr,/mob/living/carbon/human/))
 		var/mob/living/carbon/human/H = usr
 
-		H.visible_message("\red <b>[H.name]'s flesh melts right off! Holy shit!</b>")
+		H.visible_message("<span class='warning'><b>[H.name]'s flesh melts right off! Holy shit!</b></span>")
```
Comment (TheDZD) [inline code/game/dna/genes/goon_disabilities.dm]: Danger, although this is commented out code so it doesn't really matter. Same thing below.

File: code/game/dna/genes/goon_powers.dm [hunk 8]
```diff
@@ -407,7 +407,7 @@
 		if(usr.pinned.len)
 			failure = 1
 
-		usr.visible_message("\red <b>[usr.name]</b> takes a huge leap!")
+		usr.visible_message("<span class='warning'><b>[usr.name]</b> takes a huge leap!</span>")
```
Comment (TheDZD) [inline code/game/dna/genes/goon_powers.dm]: Danger

File: code/game/dna/genes/goon_powers.dm [hunk 9]
```diff
@@ -426,7 +426,7 @@
 			sleep(1)
 
 		if (FAT in usr.mutations && prob(66))
-			usr.visible_message("\red <b>[usr.name]</b> crashes due to their heavy weight!")
+			usr.visible_message("<span class='warning'><b>[usr.name]</b> crashes due to their heavy weight!</span>")
```
Comment (TheDZD) [inline code/game/dna/genes/goon_powers.dm]: Danger

File: code/game/dna/genes/goon_powers.dm [hunk 10]
```diff
@@ -435,10 +435,10 @@
 
 	if (istype(usr.loc,/obj/))
 		var/obj/container = usr.loc
-		usr << "\red You leap and slam your head against the inside of [container]! Ouch!"
+		usr << "<span class='warning'>You leap and slam your head against the inside of [container]! Ouch!</span>"
 		usr.AdjustParalysis(3)
 		usr.AdjustWeakened(5)
-		container.visible_message("\red <b>[usr.loc]</b> emits a loud thump and rattles a bit.")
+		container.visible_message("<span class='warning'><b>[usr.loc]</b> emits a loud thump and rattles a bit.</span>")
```
Comment (TheDZD) [inline code/game/dna/genes/goon_powers.dm]: Danger

File: code/game/dna/genes/goon_powers.dm [hunk 11]
```diff
@@ -580,33 +580,33 @@
 
 		switch(pain_condition)
 			if (81 to INFINITY)
-				usr << "\blue <b>Condition</b>: [M.name] feels good."
+				usr << "<span class='notice'><b>Condition</b>: [M.name] feels good.</span>"
```
Comment (TheDZD) [inline code/game/dna/genes/goon_powers.dm]: Boldnotice, same for the ones below.

File: code/game/gamemodes/changeling/changeling.dm [hunk 12]
```diff
@@ -134,8 +134,8 @@ var/list/possible_changeling_IDs = list("Alpha","Beta","Gamma","Delta","Epsilon"
 
 /datum/game_mode/proc/greet_changeling(var/datum/mind/changeling, var/you_are=1)
 	if (you_are)
-		changeling.current << "<B>\red You are a changeling!</B>"
-	changeling.current << "<b>\red Use say \":g message\" to communicate with your fellow changelings. Remember: you get all of their absorbed DNA if you absorb them.</b>"
+		changeling.current << "<B><span class='warning'>You are a changeling!</B></span>"
+	changeling.current << "<b><span class='warning'>Use say \":g message\" to communicate with your fellow changelings. Remember: you get all of their absorbed DNA if you absorb them.</b></span>"
```
Comment (TheDZD) [inline code/game/gamemodes/changeling/changeling.dm]: Danger

File: code/game/gamemodes/cult/cult.dm [hunk 13]
```diff
@@ -199,7 +199,7 @@
 /datum/game_mode/proc/remove_cultist(datum/mind/cult_mind, show_message = 1)
 	if(cult_mind in cult)
 		cult -= cult_mind
-		cult_mind.current << "\red <FONT size = 3><B>An unfamiliar white light flashes through your mind, cleansing the taint of the dark-one and the memories of your time as his servant with it.</B></FONT>"
+		cult_mind.current << "<span class='warning'><FONT size = 3><B>An unfamiliar white light flashes through your mind, cleansing the taint of the dark-one and the memories of your time as his servant with it.</B></FONT></span>"
```
Comment (TheDZD) [inline code/game/gamemodes/cult/cult.dm]: Userdanger

File: code/game/gamemodes/cult/cult.dm [hunk 14]
```diff
@@ -295,11 +295,11 @@
 	if(!check_cult_victory())
 		feedback_set_details("round_end_result","win - cult win")
 		feedback_set("round_end_result",acolytes_survived)
-		world << "\red <FONT size = 3><B> The cult wins! It has succeeded in serving its dark masters!</B></FONT>"
+		world << "<span class='warning'><FONT size = 3><B> The cult wins! It has succeeded in serving its dark masters!</B></FONT></span>"
```
Comment (TheDZD) [inline code/game/gamemodes/cult/cult.dm]: Userdanger

File: code/game/gamemodes/cult/cult.dm [hunk 15]
```diff
@@ -295,11 +295,11 @@
 	if(!check_cult_victory())
 		feedback_set_details("round_end_result","win - cult win")
 		feedback_set("round_end_result",acolytes_survived)
-		world << "\red <FONT size = 3><B> The cult wins! It has succeeded in serving its dark masters!</B></FONT>"
+		world << "<span class='warning'><FONT size = 3><B> The cult wins! It has succeeded in serving its dark masters!</B></FONT></span>"
 	else
 		feedback_set_details("round_end_result","loss - staff stopped the cult")
 		feedback_set("round_end_result",acolytes_survived)
-		world << "\red <FONT size = 3><B> The staff managed to stop the cult!</B></FONT>"
+		world << "<span class='warning'><FONT size = 3><B> The staff managed to stop the cult!</B></FONT></span>"
```
Comment (TheDZD) [inline code/game/gamemodes/cult/cult.dm]: Userdanger

File: code/game/gamemodes/cult/ritual.dm [hunk 16]
```diff
@@ -355,8 +355,8 @@ var/engwords = list("travel", "blood", "join", "hell", "destroy", "technology",
 			return
 		M.take_organ_damage(0,rand(5,20)) //really lucky - 5 hits for a crit
 		for(var/mob/O in viewers(M, null))
-			O.show_message(text("\red <B>[] beats [] with the arcane tome!</B>", user, M), 1)
-		M << "\red You feel searing heat inside!"
+			O.show_message(text("<span class='warning'><B>[] beats [] with the arcane tome!</B></span>", user, M), 1)
```
Comment (TheDZD) [inline code/game/gamemodes/cult/ritual.dm]: Danger

**Comment by TheDZD** [comment]
There's still a lot of use of <b> and <FONT size = 3> instead of using the danger, boldnotice, and userdanger spans. I would continue pointing them out, but there are at least 227 instances there, and my browser crashed from scrolling through the sheer amount of diffs.

File: code/datums/mind.dm (hunk 6 above)
Comment (tigercat2000) [inline code/datums/mind.dm]: minus the font and b tags

File: code/datums/mind.dm (hunk 4 above)
Comment (tigercat2000) [inline code/datums/mind.dm]: minus the font and b tags

**Comment by monster860** [comment]
Wow, almost every instance of danger you mentioned is completely missing a closing </b> tag.

This codebase is fucked up.

File: code/game/dna/genes/goon_powers.dm (hunk 9 above)
Comment (Crazylemon64) [inline code/game/dna/genes/goon_powers.dm]: Only the user's name is bolded here.

**Pull request closed without merging**